src/Runner.Sdk/Util/VssUtil.cs

```diff
@@ -62,11 +62,6 @@ public static void InitializeVssClientSettings(List<ProductInfoHeaderValue> addi
                 settings.SendTimeout = TimeSpan.FromSeconds(Math.Min(Math.Max(httpRequestTimeoutSeconds, 100), 1200));
             }
 
-            if (StringUtil.ConvertToBoolean(Environment.GetEnvironmentVariable("USE_BROKER_FLOW")))
```

Comment:
Should we also remove in the Runner.Listener?

https://github.com/actions/runner/blob/3f3d9b0d99755c1d329c01328f9eab26a955be65/src/Runner.Listener/MessageListener.cs#L186

Reply:
yep. missed that